content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -314,6 +314,81 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl
+{{< tabs name="other_kubectl_uninstall" >}}
+
+{{% tab name="curl" %}}
+If you have used curl to install kubectl, then the uinstallation process can be done using :-
+
+```shell
+sudo rm -rf /usr/local/bin/kubectl
+sudo rm -rf /usr/local/lib/kubectl
+```
```

Comment:
Instead of providing the predefined path, we can apply a similar approach as we did for providing instructions for [macOS](https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/#uninstall-kubectl-on-macos), by just specifying the uninstall commands as "sudo rm $(where kubectl)"